skops/card/_templates.py

```diff
@@ -0,0 +1,144 @@
+"""Templates for model cards"""
+
+from enum import Enum
+
+
+class Templates(Enum):
+    skops = "skops"
+    hub = "hub"
+
+
+CONTENT_PLACEHOLDER = "[More Information Needed]"
+"""When there is a section but no content, show this"""
+
+# fmt: off
+SKOPS_TEMPLATE = {
+    "Model description": CONTENT_PLACEHOLDER,
+    "Model description/Intended uses & limitations": CONTENT_PLACEHOLDER,
+    "Model description/Training Procedure": "",
+    "Model description/Training Procedure/Hyperparameters": CONTENT_PLACEHOLDER,
+    "Model description/Training Procedure/Model Plot": CONTENT_PLACEHOLDER,
+    "Model description/Evaluation Results": CONTENT_PLACEHOLDER,
+    "How to Get Started with the Model": CONTENT_PLACEHOLDER,
+    "Model Card Authors": (
+        f"This model card is written by following authors:\n\n{CONTENT_PLACEHOLDER}"
+    ),
+    "Model Card Contact": (
+        "You can contact the model card authors through following channels:\n"
+        f"{CONTENT_PLACEHOLDER}"
+    ),
+    "Citation": (
+        "Below you can find information related to citation.\n\n**BibTeX:**\n```\n"
+        f"{CONTENT_PLACEHOLDER}\n```"
+    ),
+}
+
+HUB_TEMPLATE = {
```

Comment:
spending more time on this, I don't see an easy way to make it suitable for usual tabular usecases. For instance, the environmental impact is pretty much zero for most models.

However, one can do NLP/image usecases and have massive models as a part of a scikit-learn pipeline. We can start with a minimal model card template and improve it then.

Reply:
So what is your suggestion here? To trim the `HUB_TEMPLATE` down to only contain those sections that make most sense for sklearn models? Or create a new template?

Reply:
my suggestion is to go back to support only one template, the one we already have, and change it/improve it/add more templates in separate PRs.

Reply:
That all sounds good to me. Supporting custom templates via dict also makes sense.